<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="dashboard_granos-20240502181034" tests="2" file=".py" time="49.635" timestamp="2024-05-02T18:11:24" failures="0" errors="2" skipped="0">
	<testcase classname="dashboard_granos" name="test_dashboard_grain" time="0.000" timestamp="0001-01-01T00:00:00" file="DashboarGranos.py" line="23">
		<error type="NoSuchWindowException" message="Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.93)
Stacktrace:
	GetHandleVerifier [0x00007FF732331502+60802]
	(No symbol) [0x00007FF7322AAC02]
	(No symbol) [0x00007FF732167CE4]
	(No symbol) [0x00007FF73213DFDF]
	(No symbol) [0x00007FF7321E1E57]
	(No symbol) [0x00007FF7321F98D1]
	(No symbol) [0x00007FF7321DA923]
	(No symbol) [0x00007FF7321A8FEC]
	(No symbol) [0x00007FF7321A9C21]
	GetHandleVerifier [0x00007FF73263411D+3217821]
	GetHandleVerifier [0x00007FF7326760B7+3488055]
	GetHandleVerifier [0x00007FF73266F03F+3459263]
	GetHandleVerifier [0x00007FF7323EB846+823494]
	(No symbol) [0x00007FF7322B5F9F]
	(No symbol) [0x00007FF7322B0EC4]
	(No symbol) [0x00007FF7322B1052]
	(No symbol) [0x00007FF7322A18A4]
	BaseThreadInitThunk [0x00007FFC92DE257D+29]
	RtlUserThreadStart [0x00007FFC9438AA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\DashboarGranos.py", line 61, in test_dashboard_grain
    find_elements(self.driver,appliy_filter)
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\Elements.py", line 19, in find_elements
    select_nex_button = WebDriverWait(driver, 10).until(
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\wait.py", line 96, in until
    value = method(self._driver)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\support\expected_conditions.py", line 363, in _predicate
    target = driver.find_element(*target)  # grab element at locator
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 742, in find_element
    return self.execute(Command.FIND_ELEMENT, {"using": by, "value": value})["value"]
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.NoSuchWindowException: Message: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=124.0.6367.93)
Stacktrace:
	GetHandleVerifier [0x00007FF732331502+60802]
	(No symbol) [0x00007FF7322AAC02]
	(No symbol) [0x00007FF732167CE4]
	(No symbol) [0x00007FF73213DFDF]
	(No symbol) [0x00007FF7321E1E57]
	(No symbol) [0x00007FF7321F98D1]
	(No symbol) [0x00007FF7321DA923]
	(No symbol) [0x00007FF7321A8FEC]
	(No symbol) [0x00007FF7321A9C21]
	GetHandleVerifier [0x00007FF73263411D+3217821]
	GetHandleVerifier [0x00007FF7326760B7+3488055]
	GetHandleVerifier [0x00007FF73266F03F+3459263]
	GetHandleVerifier [0x00007FF7323EB846+823494]
	(No symbol) [0x00007FF7322B5F9F]
	(No symbol) [0x00007FF7322B0EC4]
	(No symbol) [0x00007FF7322B1052]
	(No symbol) [0x00007FF7322A18A4]
	BaseThreadInitThunk [0x00007FFC92DE257D+29]
	RtlUserThreadStart [0x00007FFC9438AA48+40]

]]></error>
		<system-out><![CDATA[La fecha anterior no está disponible en el calendario.
]]></system-out>
	</testcase>
	<testcase classname="dashboard_granos" name="test_dashboard_grain" time="49.635" timestamp="2024-05-02T18:11:24" file="DashboarGranos.py" line="23">
		<error type="WebDriverException" message="Message: disconnected: not connected to DevTools
  (failed to check if window was closed: disconnected: not connected to DevTools)
  (Session info: chrome=124.0.6367.93)
Stacktrace:
	GetHandleVerifier [0x00007FF732331502+60802]
	(No symbol) [0x00007FF7322AAC02]
	(No symbol) [0x00007FF732167CE4]
	(No symbol) [0x00007FF73214E894]
	(No symbol) [0x00007FF73214F3FF]
	(No symbol) [0x00007FF7321682DD]
	(No symbol) [0x00007FF73213DACC]
	(No symbol) [0x00007FF7321E8F74]
	(No symbol) [0x00007FF7321DAB40]
	(No symbol) [0x00007FF7321A8FEC]
	(No symbol) [0x00007FF7321A9C21]
	GetHandleVerifier [0x00007FF73263411D+3217821]
	GetHandleVerifier [0x00007FF7326760B7+3488055]
	GetHandleVerifier [0x00007FF73266F03F+3459263]
	GetHandleVerifier [0x00007FF7323EB846+823494]
	(No symbol) [0x00007FF7322B5F9F]
	(No symbol) [0x00007FF7322B0EC4]
	(No symbol) [0x00007FF7322B1052]
	(No symbol) [0x00007FF7322A18A4]
	BaseThreadInitThunk [0x00007FFC92DE257D+29]
	RtlUserThreadStart [0x00007FFC9438AA48+40]
"><![CDATA[Traceback (most recent call last):
  File "c:\Users\luist\repos\Automatizaciones-SILOHUB-\DashboarGranos.py", line 69, in tearDown
    self.driver.close()
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 459, in close
    self.execute(Command.CLOSE)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\webdriver.py", line 348, in execute
    self.error_handler.check_response(response)
  File "C:\Users\luist\AppData\Roaming\Python\Python310\site-packages\selenium\webdriver\remote\errorhandler.py", line 229, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: disconnected: not connected to DevTools
  (failed to check if window was closed: disconnected: not connected to DevTools)
  (Session info: chrome=124.0.6367.93)
Stacktrace:
	GetHandleVerifier [0x00007FF732331502+60802]
	(No symbol) [0x00007FF7322AAC02]
	(No symbol) [0x00007FF732167CE4]
	(No symbol) [0x00007FF73214E894]
	(No symbol) [0x00007FF73214F3FF]
	(No symbol) [0x00007FF7321682DD]
	(No symbol) [0x00007FF73213DACC]
	(No symbol) [0x00007FF7321E8F74]
	(No symbol) [0x00007FF7321DAB40]
	(No symbol) [0x00007FF7321A8FEC]
	(No symbol) [0x00007FF7321A9C21]
	GetHandleVerifier [0x00007FF73263411D+3217821]
	GetHandleVerifier [0x00007FF7326760B7+3488055]
	GetHandleVerifier [0x00007FF73266F03F+3459263]
	GetHandleVerifier [0x00007FF7323EB846+823494]
	(No symbol) [0x00007FF7322B5F9F]
	(No symbol) [0x00007FF7322B0EC4]
	(No symbol) [0x00007FF7322B1052]
	(No symbol) [0x00007FF7322A18A4]
	BaseThreadInitThunk [0x00007FFC92DE257D+29]
	RtlUserThreadStart [0x00007FFC9438AA48+40]

]]></error>
		<system-out><![CDATA[La fecha anterior no está disponible en el calendario.
]]></system-out>
	</testcase>
</testsuite>
